doc/platforms/cray.rst

```diff
@@ -159,7 +160,8 @@ Building Chapel for a Cray System from Source
    :ref:`readme-chplenv`.
 
      :``CHPL_TASKS``: tasking implementation, default ``qthreads``
-     :``CHPL_COMM``: communication implementation, default ``gasnet``
+     :``CHPL_COMM``: communication implementation, default ``ugni`` on
+                     XC/XE systems, ``gasnet`` on CS systems
```

Comment:
Yeah, I went with XE/XC originally, but changed to be consistent with the rest of the file. For now I'd prefer to leave it as it is and revamp/simplify this readme once muxed is retired.